specs/openapi.yml

```diff
@@ -0,0 +1,199 @@
+openapi: 3.0.3
+info:
+  title: Anoma
+  description: Interacting with an Anoma blockchain via Tendermint RPC
+  version: 0.6.1
+servers:
+  - url: http://127.0.0.1:26657
+    description: Tendermint RPC endpoint for an Anoma ledger
+paths:
+  /:
+    post:
+      summary: Interact with the Anoma blockchain via Tendermint RPC
+      operationId: abci_query
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  description: Should be unique between requests
+                  type: integer
+                  example: 58392
+                method:
+                  description: The Tendermint RPC method being called which in this case should always be abci_query
+                  type: string
+                  enum:
+                    - "abci_query"
+                params:
+                  type: object
+                  required:
+                    - path
+                  properties:
+                    path:
+                      description: Path as will be recognized by the ledger
+                      oneOf:
+                        - type: string
+                          enum:
+                            - "epoch"
```

Comment:
Docstrings for these values taken from https://github.com/anoma/anoma/blob/9475cb338489b37aa44971ef4d768e2a57b20aa5/apps/src/lib/node/ledger/rpc.rs#L16 , although for "epoch" I think we can query the epoch for any given block height rather than just the latest block, when going via `?abci_query`.

Reply:
I've opened https://github.com/anoma/namada/issues/172